model/src/main/scala/hmda/model/institution/Institution.scala

```diff
@@ -10,6 +10,7 @@ case class Institution(
     agency: Agency,
     institutionType: InstitutionType,
     hasParent: Boolean,
+    cra: Boolean = false, // TODO do we have this info when creating the institution? if so, then don't default here.
```

Comment:
I think the answer is _yes_.  CRA is a separate data source, but  we should be able to merge it with NIC whenever creating/updating Institution records.